ci-operator/step-registry/upi/deprovision/heterogeneous/powervs/upi-deprovision-heterogeneous-powervs-commands.sh

```diff
@@ -91,14 +88,23 @@ function cleanup_ibmcloud_powervs() {
     done
   done
 
+  # Delete any vpc older than 24 hrs
+  echo "Cleaning up VPCs"
+  for VPC in $(ibmcloud is vpcs --resource-group-name "${RESOURCE_GROUP}" 2>&1 | grep "${RESOURCE_GROUP}" | grep - v Listing | grep -i ci-op | awk '{print $1}')
+  do
+    echo "VPC=${VPC}"
+    ibmcloud is vpc-delete "${VPC}" -f
+    sleep 20s
+    echo "Done cleaning up VPCs"
+  done
```

Comment:
We would need to do some fancy checking as the next ipi-deprovision step will fail. Can you do this in the next PR?

```suggestion
```